gpflow/inducing_variables/multioutput/inducing_variables.py

```diff
@@ -129,7 +129,7 @@ class FallbackSeparateIndependentInducingVariables(MultioutputInducingVariables)
     """
 
     @check_shapes(
-        "inducing_variable_list[all]: [M, D, 1]",
+        "inducing_variable_list[all]: [., D, 1]",
```

Comment:
(have you been preferring `.` to `None` then?)

Reply:
> (have you been preferring `.` to `None` then?)

Yes. I think `None` is weird, but some people like it, and I aim to please.